app/models/repair.rb

```diff
@@ -4,9 +4,14 @@ class Repair < ActiveRecord::Base
                   :status, :user_id, :cost
   belongs_to :item
   belongs_to :person
-  validates_numericality_of :cost
+  delegate :item_category, :to => :item
+  delegate :item_type, :to => :item
```

Comment:
I need these for the bread crumbs, so as not to have a lot of logic in the view. This is what we discussed in person as well.